.github/workflows/auto-build-rpm.yml

```diff
@@ -49,7 +56,7 @@ jobs:
         run: |
           export VERSION=${{ steps.branch_env.outputs.version }}
           sudo gem install --no-document fpm
-          git clone https://github.com/api7/apisix-build-tools.git
+          git clone https://github.com/FangSen9000/apisix-build-tools.git
```

Comment:
Note: This needs to be changed before merging. I link to the build script that I have changed. And we must let the API7 community through my changes and upgrades in them to change back.